pkg/api/types.go

```diff
@@ -28,6 +28,53 @@ type Project struct {
 	UpdatedTime time.Time `bson:"updatedTime,omitempty" json:"updatedTime,omitempty" description:"updated time of the project"`
 }
 
+// PipelineRecord is all information about some pipeline building history.
```

Comment:
Move definition of `PipelineRecord` after `Pipeline` as `PipelineRecord` depends on `Pipeline`.